app/commands/github/issue/open.rb

```diff
@@ -0,0 +1,41 @@
+module Github
+  class Issue
+    class Open
+      include Mandate
+
+      initialize_with :repo, :title, :body
+
+      def call
+        if missing?
+          create_issue
+        elsif closed?
+          reopen_issue
+        end
```

Comment:
There is a risk of duplicate issues being created if this is called in parallel. Do we want to guard against that?